.travis.yml

```diff
@@ -36,30 +36,6 @@ jobs:
       script:
         - vendor/bin/phpunit
 
-    - <<: *TEST
-
-      php: 7.0
-
-      env: WITH_LOCKED=true
-
-    - <<: *TEST
-
-      php: 7.0
-
-      env: WITH_HIGHEST=true
-
-    - <<: *TEST
-
-      php: 7.1
-
-      env: WITH_LOWEST=true
-
-    - <<: *TEST
-
-      php: 7.1
-
-      env: WITH_LOCKED=true
```

Comment:
I've removed this one because the `composer.lock` file is generated under PHP 7.2, so it should be used only under the same platform.

Reply:
@Jean85 

Ha, I think it would be best to check in the version of `composer.lock` as generated on the lowest supported version, what do you think?

Reply:
No opinions about that for me! I would leave it to @Jan0707

Reply:
Still not sure whether to actually check it in at all. I did initially check it in to make sure all my dev machines were running the same versions of dependencies, but I also see the downsides.

Reply:
I would leave it like that and postpone the decision then, it's better if we release this PR ASAP and measure any downside in practice, if any.